src/ctia/import/threatgrid/http.clj

```diff
@@ -0,0 +1,66 @@
+(ns ctia.import.threatgrid.http
+  (:require [clj-http.client :as http]
+            [clj-http.conn-mgr :as conn]
+            [clojure.core.async :as async]
+            [clojure.java.io :as io]))
+
+(defn tg-uri [feed-file-name]
+  (str "https://panacea.threatgrid.com/api/v3/feeds/" feed-file-name))
+
+;; get feed data via direct http call
+(defn feed-file-names [feed-names dates]
+  (for [feed-name feed-names
+        date-str dates]
+    (str feed-name "_" date-str ".json")))
+
+(defn body-empty? [body]
+  (or (empty? body)
+      (= "[]" body)))
+
+(defn download-feeds
+  [{:keys [feed-path feed-names dates tg-api-key] :as cli-options}]
+  (let [options {:query-params {"api_key" tg-api-key}}]
+    ;; don't download a feed if we already have it
+    (doseq [feed-file-name (filter #(not (.exists (io/as-file (str feed-path "/" %))))
+                                   (feed-file-names feed-names dates))]
+      (Thread/sleep 1000) ;; a courtesy
+      (let [body (:body (http/get (tg-uri feed-file-name) options))]
+        (if (not (body-empty? body))
+          (let [file-path (str feed-path "/" feed-file-name)]
+            (println "Downloading" file-path)
+            (with-open [wrtr (io/writer file-path)]
+              (.write wrtr body)))
+          (println feed-file-name "is empty."))))))
+
+(defn retry
```

Comment:
I agree in principle, but don't think that should be a blocking issue for this PR.